lab-justin/gulpfile.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+
+var paths = ['**/*.js', './lib/*.js'];
+
+gulp.task('eslint', function(){
+  return gulp.src(paths)
+  .pipe(eslint())
+  .pipe(eslint.format())
+  .pipe(eslint.failAfterError());
+});
+
+gulp.watch('watch', ()=>{
+  gulp.watch('**/*.js', !'package.json', ['eslint']);
```

Comment:
Anything ending in .json will already be excluded, because it's only looking for stuff ending with .js